internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubTopicImpl.java

```diff
@@ -42,7 +42,7 @@ public boolean equals(Object o) {
     if (this == o) {
       return true;
     }
-    if (o == null || !(o instanceof PubSubTopic)) {
```

Comment:
Why do we remove the null check?

Reply:
It's redundant in this case as `instanceof` returns false if `o` is null